web/src/components/pages/home/Node.tsx

```diff
@@ -0,0 +1,27 @@
+export default function Node() {
+
+    function genCol() {
+        const col = [];
+
+        for (let i = 0; i < 15; i++) {
+            col.push(
+                <button key={i} style={{ backgroundColor: 'yellow', outline: '1px solid black', width: '40px', height: '40px' }}></button>
```

Comment:
Could we turn the native tags into Mantine components and move the button styles into a variable?

Reply:
The reason I choose the native button tag over the library one because this is technically not a button in our application but rather a vertex or node for our graph visualization. Considering that the button from mantine might restrict our future customization, I decided that the native will be the best option. For the style: I will make it a variable instead as you suggested. Open for idea on this.

Reply:
If this is the case, should \<div> be used instead of \<button>?

Reply:
Make sense, I just don't want to write additional style for div to not span the entire width. But if you think <div> is a better choice, I can go with that. Let me know what you think.

Reply:
Do the nodes need to be interactive?

Reply:
Yes, we might need it to be.

Reply:
Okay, well in that case, having a button would probably be the best approach.

Also, is there any way we can make the visualizer responsive for mobile?

Reply:
It's good to make it responsible. However, we'll start small. I think we can focus on this feature after we able to get everything done first. Just my idea, what's your?